ui-service/src/utils/UserData.js

```diff
@@ -14,8 +14,24 @@ const gender = [
   { value: 3, name: "No binario" },
 ];
 
+const months = [
```

Comment:
For this you can use the js API, I leave an eg:
`const date = new Date(2022, 10, 25);`
`const month = date.toLocaleString('default', { month: 'long' });`
`console.log(month);`

This will to print -> Octubre or October